arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -4,11 +4,55 @@
     "version": "2016-06-27-preview",
     "title": "ContainerRegistry"
   },
-  "host": "https://management.azure.com",
+  "host": "management.azure.com",
   "schemes": [
-    "http"
+    "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/moveResources": {
+      "post": {
+        "tags": [
+          "Operation"
+        ],
+        "operationId": "Operation_MoveResources",
+        "consumes": [
+          "application/json",
+         "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroup",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "moveRequest",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RegistryMoveRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        },
+        "deprecated": false
+      }
+    },
     "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroup}/providers/Microsoft.ContainerRegistry/registries/{registryName}": {
       "get": {

```

Comment:
Please add descriptions
get: "Returns the properties for the specified container registry."
resourceGroup: "The name of the resource group within the user's subscription."
registryName: "The name of the container registry."